totalContributors.py

```diff
@@ -0,0 +1,83 @@
+import subprocess
+import pathlib
+import os.path
+import sys
+import datetime
+
+def add_log(dict, input):
+    for dataset in input:
+        name = dataset.split("\t")[1]
+        value = int(dataset.split("\t")[0])
+        if name in dict:
+            dict[name] += value
+        else:
+            dict[name] = value
+
+def retrieve_log(repo, version):
+    return subprocess.run(['git', 'shortlog', '-s', version], capture_output = True, text = True, 
+        cwd = os.path.dirname(pathlib.Path().resolve()) + repo).stdout.splitlines()
+
+def find_version(time, repo):
+    versions_list = subprocess.run(['git', 'tag', '-l'], capture_output = True, text = True, 
+        cwd = os.path.dirname(pathlib.Path().resolve()) + repo).stdout.splitlines()
+    for version in versions_list:
+        version_time = subprocess.run(['git', 'log', '-1', '--format=%ai', version], capture_output = True, text = True, 
+            cwd = os.path.dirname(pathlib.Path().resolve()) + repo).stdout.split()[0]
+        if(create_Time_Object(version_time) >= create_Time_Object(time)):
+            return version
+    return version
+
+def create_Time_Object(time):
```

Comment:
I think the idiomatic way to implement this is by using `datetime.strptime`. See e.g. https://stackoverflow.com/questions/466345/converting-string-into-datetime (you don't have to specify the hour and minute).